packages/cli/src/commands/org/create.ts

```diff
@@ -0,0 +1,50 @@
+import {Command, flags} from '@oclif/command';
+import {cli} from 'cli-ux';
+import {Config} from '../../lib/config/config';
+import {
+  Preconditions,
+  HasAccessToken,
+} from '../../lib/decorators/preconditions/';
+import {AuthenticatedClient} from '../../lib/platform/authenticatedClient';
+
+export default class OrgCreate extends Command {
+  static description = 'Create a new trial Coveo organization';
+
+  private configuration!: Config;
+
+  static flags = {
+    setDefaultOrganization: flags.boolean({
+      char: 'D',
+      default: false,
+      description: 'Set the newly created organization as the default one',
+    }),
+  };
+
+  static args = [
+    {
+      name: 'name',
+      description: 'The name to assign to the new organization.',
+      required: true,
+    },
+  ];
+
+  @Preconditions(HasAccessToken())
```

Comment:
Similar comment as above: It's not because you have an access token in a local configuration file that it will work.

You still need to verify that the token is valid.

If what you are trying to do is to `verify the validity of an access token without requiring the existence/knowledge of an organization ID`, then we need to change this in either platform-client ( https://github.com/coveo/platform-client ), or to add a function in authenticated client to call `platform.cloud.coveo.com/oauth/check_token`